vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/ExhibitionLanguage.java

```diff
@@ -20,6 +25,7 @@ public class ExhibitionLanguage extends VSpaceElement implements IExhibitionLang
     private String label;
     
     @ManyToOne(targetEntity = Exhibition.class)
+    @JsonManagedReference()
```

Comment:
are you sure this is correct and it doesn't have to be the other way (exhibition includes exhibitionLanguage)?